resources/carrier/en/234.txt

```diff
@@ -151,7 +155,13 @@
 2347028|Starcomms
 2347029|Starcomms
 234703|MTN
-234704|Visafone
+2347040|Visafone
```

Comment:
@penmetsaa The ITU doc references the NCC website for getting the info. Visafone is not assigned the entire range. Please confirm this [http://www.ncc.gov.ng/technology/standards/numbering#mobile-telephony-number-allocation](url)